queries/scfg/highlights.scm

```diff
@@ -0,0 +1,8 @@
+[
+ "{"
+ "}"
+ ] @punctuation.bracket
+
+(comment) @comment @spell
```

Comment:
You probably want to add an injection for the comment parser, too.

Reply:
Done